src/coreclr/nativeaot/Runtime/RhConfig.cpp

```diff
@@ -179,6 +191,103 @@ void RhConfig::ReadEmbeddedSettings()
     return;
 }
 
+bool RhConfig::GetEmbeddedKnob(_In_z_ const char* configName, _Out_ const char** configValue)
+{
+    // Read the config if we haven't yet
+    if (g_embeddedKnobs == NULL)
+    {
+        ReadEmbeddedKnobs();
+    }
+
+    // Config wasn't read or reading failed
+    if (g_embeddedKnobs == CONFIG_INI_NOT_AVAIL)
+    {
+        return false;
+    }
+
+    const ConfigPair* configPairs = (const ConfigPair*)g_embeddedKnobs;
+
+    // Find the first name which matches (case insensitive to be compat with environment variable counterpart)
```

Comment:
TODO: compare with coreclr - is this case insensitive?

Reply:
Whoops, that TODO was for me, but might as well dump it on you :). Can you compare how this is done in CoreCLR?